### PR TITLE
set 443 as default port

### PR DESCRIPTION
**Pull request by huguesBouvier**
set 443 as default port

### CODE REVIEW

File: templates/authorization/deploymentTopLayer.json [hunk 1]
```diff
@@ -21,12 +21,12 @@
                     "IoTEdgeAPIProxy": {
                         "settings": {
                             "image": "mcr.microsoft.com/azureiotedge-api-proxy:1.0",
-                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"8000/tcp\": [{\"HostPort\":\"8000\"}]}}}"
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"443/tcp\": [{\"HostPort\":\"443\"}]}}}"
```
Comment (varunpuranik) [inline templates/authorization/deploymentTopLayer.json:24]: Why is the port mapping needed on registry module?

Comment (huguesBouvier) [reply templates/authorization/deploymentTopLayer.json:24]: What do you mean? This is API proxy.
Registry module is still 5000

Comment (huguesBouvier) [reply templates/authorization/deploymentTopLayer.json:24]: oh I see, why is it map to 5000? It is not needed. I suppose it was added as a convenience. 
It is not ISA95 so it has no impact here.

Comment (huguesBouvier) [reply templates/authorization/deploymentTopLayer.json:24]: ok I can remove the mapping. To be perfectly consistent we need to remove it in the IIOT report too.

**Pull request merged**